examples/usage/cooldown.rs

```diff
@@ -0,0 +1,163 @@
+//! Demonstrates implementing a cooldown in UI.
+//! We create three buttons with 2, 1, and 5 seconds cooldown.
+
+use bevy::{
+    color::palettes::tailwind::{SLATE_400, SLATE_50},
+    ecs::spawn::SpawnIter,
+    prelude::*,
+};
+
+fn main() {
```

Comment:
I think it would be nice to have a bit of inline code comments in this example. It's not needed for the UI stuff that's not strictly needed for a cooldown system, but I think it would be nice to draw attention to the critical code parts and explain them a bit more